src/kvstore/listener/test/NebulaListenerTest.cpp

```diff
@@ -6,6 +6,8 @@
 #include <gtest/gtest.h>
 #include <thrift/lib/cpp/concurrency/ThreadManager.h>
 
+#include <map>
```

Comment:
It seems you don't need to modify this test since derived Listener do not need to overload `apply` function anymore. Not a big issue, up to you.

Reply:
Just to compile and pass the test